homeassistant/components/homekit_controller/binary_sensor.py

```diff
@@ -2,35 +2,15 @@
 import logging
 
 from homeassistant.components.binary_sensor import BinarySensorDevice
+from homekit.model.characteristics import CharacteristicsTypes
```

Comment:
Please group imports according to PEP8.
https://www.python.org/dev/peps/pep-0008/#imports

Move this line between logging and homeassistant.